addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/PiiScanRuleUnitTest.java

```diff
@@ -44,6 +48,11 @@ protected PiiScanRule createScanner() {
         return new PiiScanRule();
     }
 
+    @BeforeEach
+    private void resetThreshold() {
+        rule.setAlertThreshold(AlertThreshold.MEDIUM);
```

Comment:
This could be done when the scan rule is created (iirc, that's where other tests do the scan rule setup too).